### PR TITLE
Quickstart docs

### PR DESCRIPTION
**Pull request by njgheorghita**
## What was wrong?
quickstart docs to provide a simple jumping-in point for new users.

#### Cute Animal Picture
image

### CODE REVIEW

**Comment by njgheorghita** [comment]
@corydickson If you have some spare time to give this a quick look through for any errors / confusing lines of thought - that'd be awesome!

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+   ethpm auth --keyfile-path KEYFILE_PATH
+
+Test that your keyfile has been properly stored.
+
+.. code-block:: bash
+
+   ethpm auth
+   > Keyfile stored for address: 0x123abc....................
+
+You can now use your keyfile's password as the flag ``--keyfile-password`` for any ethPM command, and it will be used to automatically sign any transactions.
```
Comment (corydickson) [inline docs/quickstart.rst]: Nit: change 'as the flag' -> 'with the flag' as it sounds like your password could be used as an argument by typing `--[my-password]`

File: docs/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+If you have the Solidity compiler installed on your machine, the best way to get started is with the manifest wizard. The wizard expects a project directory with the following structure. 
+
+- project/
+  - contracts/
+    - xxx.sol
+    - yyy.sol
+
+Pass in a path to your project directory under the ``--project-dir`` flag. The wizard will attempt to compile these contracts using the available ``solc`` on your machine. Then it will prompt you for package details.
+
```
Comment (corydickson) [inline docs/quickstart.rst:113]: Do you think it's worth documenting a minimum required solc version? This way a user knows that the package they want to release must use the same pragma as the `registry` contract if they are to deploy their own.

Comment (njgheorghita) [reply docs/quickstart.rst:113]: Good call. Just to clarify though, the minimum solc required version would just be whatever version is required to compile the contracts that live in the project directory - not the pragma required for the registry contract. The registry contract is already compiled and deployed to the blockchain - to release a package on the registry all you need is the `package_name`, `version` and `manifest_uri`

**Review by pipermerriam — APPROVED** [review]
Nice!

**Pull request merged**